README.md

```diff
@@ -63,6 +63,10 @@ Options:
                     - <file>
                     NOTE: only both emulated, or both real.
                           mixing not possible
+  -ataid0=<mode>    ATA 0 HDD ID emulation mode, supported are:
```

Comment:
I think `ata0id` instead of `ataid0` is better becouse it's the `id` of `ata0`.

Reply:
done